src/components/ShiftAbsenceShow.tsx

```diff
@@ -3,7 +3,7 @@ import { GetAbsenceForAdmin200ResponseData as Absence } from "@/api/typescript-a
 
 // TODO: shift 側を修正した時に、型も調整
 type ShiftAbsenceProps = {
-  fetchData: Absence;
+  fetchData?: Absence;
```

Comment:
今さらですが、この変数名はすなおに absence にしたほうが読みやすいのでは？

```suggestion
  absence?: Absence;
```

Reply:
@yuto-ono 
https://github.com/tatata05/next-app/pull/22/commits/a0d0e3ad6ab4125b8cd93dfb771d245b11e763cd
修正いたしました！！
お願いいたします。